### DISCUSSION TITLE
自建onlyofficedoc如何对接

### DISCUSSION THREAD

**Discussion by wangyangaa** [libsgh/PanIndex#386] [Q&A]
你好，请问自建的onlyoffice如何对接panindex，在view-office.html中接口地址处应该如何填写？
尝试了以下地址均不行：
https://自己部署的域名/
https://自己部署的域名/web-apps/apps/api/documents/api.js
https://自己部署的域名/7.1.1-23/web-apps/apps/spreadsheeteditor/main/index.html?_dc=7.1.1-23&lang=zh-CN&customer=ONLYOFFICE&frameEditorId=office-body&mode=view&parentOrigin=

![图片](https://user-images.githubusercontent.com/50029644/211995898-c3e48062-644c-4f00-8b43-7fdc0856100b.png)

**Comment by libsgh** [comment]
有空我会再研究一下，也许不用等到升级，自定义下js就可以实现
